specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -78,3 +78,94 @@ model Evaluation {
   @doc("Evaluators to be used for the evaluation.")
   evaluators: Record<EvaluatorConfiguration>;
 }
+
+@doc("Definition for sampling strategy.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v_latest)
+model SamplingConfiguration {
```

Comment:
Does SamplingConfiguration apply to only input that comes from app insights? If so, my recommendation is to add a new polymorphic type for Input Data called "traces" or "agent_input" and move this into that polymorphic object,

Reply:
Rename `SamplingConfiguration` to `AgentEvaluationSamplingConfiguration`, do you think it's better? From the long term, `AgentEvaluationSamplingConfiguration` gates all the agent related evaluations, which outputs to AppInsights.

Reply:
Discussed offline, we will cover this by the evaluation API convergence project